action.js

```diff
@@ -414,6 +425,21 @@ async function findAsanaTaskId(){
     }
 }
 
+async function getTaskPermalink(asanaTaskId){
+    const taskId = core.getInput('github-pat', {required: true});
```

Comment:
I think this should be
```suggestion
    const taskId = core.getInput('asana-task-id', {required: true});
```